core_backend/startup.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 python -m alembic upgrade head
+python add_users_to_db.py
```

Comment:
I guess this is just in lieu of google oauth. There is no other way to create users right now?

Reply:
Yup until we have OAuth. No other way to add users right now.